### ISSUE TITLE
HAP and recog - large files

### ISSUE THREAD

**Issue by alex-konovalov** [gap-system/gap-distribution#64]
merged packages archive tar.gz increased from 317.98 to 330.59 MB after picking up [ "curlInterface", "2.1.0" ], [ "HAP", "1.12.7" ], [ "IO", "4.5.3" ],  [ "matgrp", "0.51" ],  [ "recog", "1.3.1" ], [ "Semigroups", "3.0.19" ]. 

In particular, the following appeared in the log:
```
pkg/Hap1.12/test/examples/image1.3.2.eps: up to 144 MB of RAM may be required to manage this file
(use 'hg revert pkg/Hap1.12/test/examples/image1.3.2.eps' to cancel the pending addition)
```
and
```
pkg/recog-1.3.1/examples/c3c5benchmarks_mats.g: up to 66 MB of RAM may be required to manage this file
(use 'hg revert pkg/recog-1.3.1/examples/c3c5benchmarks_mats.g' to cancel the pending addition)
```
Need to look at large files in these packages.

**Comment by alex-konovalov** [comment]
Indeed, unpacked 194M for Hap1.12 and 29M for recog-1.3.1. 

In HAP, `image1.3.2.eps` is 47MB large, and in recog `c3c5benchmarks_mats.g` uses 22MB.

To compare, in GAP 4.9.3 I have 132M for Hap1.12 and 28M for recog-1.2.5. 

For the .g file in recog, perhaps we could use .gz instead @fingolfin ? I guess this file is static and you will not change it whatsoever. I am not so sure about the purpose of the .eps file in HAP. Will ask the author.

**Comment by alex-konovalov** [comment]
Also, there should be no archives with same name and different version inside. 

Currently  http://hamilton.nuigalway.ie/Hap/www/download/downloadIndex.html offers http://hamilton.nuigalway.ie/Hap/hap1.12.tar.gz, and that's the same name of the archive but different version of the package.

**Comment by fingolfin** [comment]
@alex-konovalov I think this is fixed in recog 1.3.2 and Hap1.21 -- at least those two files are.
